configs/ci/roles/ros.json

```diff
@@ -0,0 +1,19 @@
+{
+    "roles": [
+        {
+          "name": "ROS Administrator",
+          "description": "Perform any available operation against any Resource Optimization service.",
+          "system": true,
+          "platform_default": true,
```

Comment:
Is the expected behavior to give everyone admin access by default?

Reply:
We want the app to provide Read access to every user (may be accept the administrator).

Reply:
@coderbydesign, 
As @apuntamb mentioned in case of ROS, we want to provide read access to every user. Mainly ROS backend is mainly r/o.

Any suggestions on defining a role? 
For now,  can we just go with admin role or do we need to define two roles i.e admin & user where user role will be added by default to each user?

Reply:
Ok, yes you'd need a role such as "Resource Optimization user", which would have the `ros:*:read` permission, and `platform_default: true`, vs having this administrator role be a platform defaultt.